### ISSUE TITLE
Commit your project to Git and deploy to the cloud - step 18 does not accept answer

### ISSUE THREAD

**Issue by kjetil-kilhavn** [SAPDocuments/Tutorials#3139]
Tutorial URL: https://developers.sap.com/tutorials/teched-2016-5.html

Step 18 refuses to accept that business contact ID 100000031 is indeed the right answer.
![image](https://user-images.githubusercontent.com/14242355/53368206-69627700-3948-11e9-9327-1babcc5e6f2e.png)
![image](https://user-images.githubusercontent.com/14242355/53368374-dd9d1a80-3948-11e9-9f55-7bcd7cbb8d14.png)

**Comment by thecodester** [comment]
@mhassett92 Can you take a look at this?

@kjetil-kilhavn These are quite old tutorials and it could be that the URL has changed, but I will check.

**Comment by mts2605** [comment]
Just the same problem I have. Please review the "validation";-)

**Comment by thecodester** [comment]
From where did you start doing the tutorial? I believe the tutorial assumes you've set up a project te2016. Also, the URL you posted has an app name of businesspartners but the tutorials has it as te2016 (though this isn't the only problem with the validation).

**Comment by mts2605** [comment]
@thecodester I did get the exact same url as @kjetil-kilhavn (with different S-user of course) And even if I change the URL to include "te2016" as replacement for "businesspartners", it makes no difference. I believe there must be some kind of validation failure. (Or outdated..!?)

**Comment by kjetil-kilhavn** [comment]
I am quite sure it was this https://developers.sap.com/tutorials/teched-2016-4.html

@thecodester - that was a good question :-)
One of the problems with the linked content on world wide web is exactly that, remembering where you started. I started a tutorial for WebIDE to create an application based on a master-detail template. I see now there is not only one such tutorial - but since I haven't completed very many tutorials yet I found it via my 'dashboard'.
